### ISSUE TITLE
Ads with unique sizes appear weirdly

### ISSUE THREAD

**Issue by ytcyde** [modrinth/code#2367]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [X] I have tried resolving the issue using the [support portal](https://support.modrinth.com)
- [X] I have ensured my Modrinth App installation is up to date

### What version of the Modrinth App are you using?

Modrinth App v0.8.5

### What operating systems are you seeing the problem on?

Windows

### Describe the bug

Banner ads appear with this background behind it
![image](https://r2.fakecrime.bio/uploads/819f4583-a34f-455a-b162-d7d850367192.png)
How it appears if the ad is a square
![image](https://r2.fakecrime.bio/uploads/63df861f-ce84-4a49-a1ed-36cb17cf7eaf.png)

### Steps to reproduce

1. Get lucky/unlucky and get ad that isn't a square 

### Expected behavior

The background square doesn't show.

**Comment by Prospector** [comment]
Yeah, unfortunately the ads may come in a variety of sizes. that is unfortunately out of our control.
